apidef/api_definitions.go

```diff
@@ -222,9 +222,10 @@ type HeaderInjectionMeta struct {
 }
 
 type HardTimeoutMeta struct {
-	Path    string `bson:"path" json:"path"`
-	Method  string `bson:"method" json:"method"`
-	TimeOut int    `bson:"timeout" json:"timeout"`
+	Disabled bool   `bson:"disabled" json:"disabled"`
+	Path     string `bson:"path" json:"path"`
+	Method   string `bson:"method" json:"method"`
+	TimeOut  int    `bson:"timeout" json:"timeout"`
```

Comment:
@furkansenharputlu I think `TimeOut` should be `Timeout` no capital O

Reply:
That part is already existing in the code. I just added disabled flag.